### ISSUE TITLE
Stange error messages

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-gui#1979]
I still get these strange error messages when starting GNS3 on Linux (here Ubuntu 16.04)

![error_property](https://cloud.githubusercontent.com/assets/4752614/24741964/5d613706-1a65-11e7-8589-99ca7c4ada0c.png)

**Comment by noplay** [comment]
I already saw that. It's  when a SVG symbol contain invalid property. Qt
ignore them but log the error. Their is no way to turn this off.

I fixed the symbol shipped with GNS3 but you probably have one in your
symbol directory with invalid properties.

On Thu, Apr 6, 2017 at 9:08 AM Jeremy Grossmann <notifications@github.com>
wrote:

I still get these strange error messages when starting GNS3 on Linux (here
Ubuntu 16.04)

[image: error_property]
<https://cloud.githubusercontent.com/assets/4752614/24741964/5d613706-1a65-11e7-8589-99ca7c4ada0c.png>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/GNS3/gns3-gui/issues/1979>, or mute the thread
